### PR TITLE
html css popup

### PR DESCRIPTION
**Pull request by Shrai-dev**
### HTML & CSS practice: Hooli-style Popup

[Demo](https://shrai-dev.github.io/kottans-html-css-popup/) |
[Code base](https://github.com/Shrai-dev/kottans-html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Shrai-dev/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                </li>
+                <li class="nav__menu-item"><a href="#"><img src="popup-icons/bell.png" alt="Bell" width="25" height="30"></a></li>
+                <li class="nav__menu-item"><a href="#"><img class="user" src="popup-icons/user.png" alt="Ico"
+                            width="30"></a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <img src="" alt="">
+    </main>
```
Comment (smillims) [inline submissions/Shrai-dev/html-css-popup/index.html:77]: I think that you should add way to your `img` and add description to `alt` or delete it's block

File: submissions/Shrai-dev/html-css-popup/main.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+#more-popup:checked ~ .more__popup-menu::before {
+    content: "";
+    width: 100%;
+    border-top: 2px solid #c4c4c4;
+    padding: 10px 0;
+}
+
+#more-popup:checked ~ .more__submenu, 
+#more-popup:checked {
+    display: none;
```
Comment (smillims) [inline submissions/Shrai-dev/html-css-popup/main.css:159]: Try to use just classes white you're styling

File: submissions/Shrai-dev/html-css-popup/main.css [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+}
+
+.more__submenu:hover,
+.more__submenu:focus,
+.more__popup-link:hover,
+.more__popup-link:focus {
+    cursor: pointer;
+    outline: thin solid #c2c2c2;
+    background-color:  #b3b3b3;
+}
```
Comment (smillims) [inline submissions/Shrai-dev/html-css-popup/main.css:174]: Your More `button` don't work by using just keyboard. Please fix it

**Review by smillims — COMMENTED** [review]
Hey, good work! But I find some problems here, let's improve it together

File: submissions/Shrai-dev/html-css-popup/index.html (hunk 1 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/html-css-popup/index.html]: Done.

File: submissions/Shrai-dev/html-css-popup/main.css (hunk 2 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/html-css-popup/main.css]: Done.

File: submissions/Shrai-dev/html-css-popup/main.css (hunk 3 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/html-css-popup/main.css]: Done.

File: submissions/Shrai-dev/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                                            alt="Promos">Promos</a></li>
+                                <li class="more__list"><a class="more__popup-link" href="https://github.com/kottans" target="_blank">Even more from Kottans</a>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                </li>
+                <li class="nav__menu-item"><a href="#"><img src="popup-icons/bell.png" alt="Bell" width="25" height="30"></a></li>
+                <li class="nav__menu-item"><a href="#"><img class="user" src="popup-icons/user.png" alt="Ico"
+                            width="30"></a></li>
```
Comment (lenadgit) [inline submissions/Shrai-dev/html-css-popup/index.html:71]: the same as on line 20

File: submissions/Shrai-dev/html-css-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+                                            alt="Blogs">Blogs</a></li>
+                                <li class="submenu__list-item"><a href="#"><img src="popup-icons/promo.png"
+                                            alt="Promos">Promos</a></li>
+                                <li class="more__list"><a class="more__popup-link" href="https://github.com/kottans" target="_blank">Even more from Kottans</a>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                </li>
+                <li class="nav__menu-item"><a href="#"><img src="popup-icons/bell.png" alt="Bell" width="25" height="30"></a></li>
```
Comment (lenadgit) [inline submissions/Shrai-dev/html-css-popup/index.html:69]: the same as on line 20

File: submissions/Shrai-dev/html-css-popup/main.css [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.nav__menu-item {
+    position: relative;
+    margin: 5px 20px;
+}
+
+a:hover,
```
Comment (lenadgit) [inline submissions/Shrai-dev/html-css-popup/main.css:36]: good practice - add the classes for links, images (do not add styles to the tags). Could you please fix it?

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/Shrai-dev/html-css-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+                <li class="nav__menu-item"><a href="#"><img class="user" src="popup-icons/user.png" alt="Ico"
+                            width="30"></a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
```
Comment (lenadgit) [inline submissions/Shrai-dev/html-css-popup/index.html:75]: Where are the styles for this class "main"?  What for it?

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/Shrai-dev/html-css-popup/index.html (hunk 7 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/html-css-popup/index.html]: Forgot to delete it. Now it's gone)

File: submissions/Shrai-dev/html-css-popup/main.css (hunk 6 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/html-css-popup/main.css]: Fixed.

File: submissions/Shrai-dev/html-css-popup/index.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="shortcut icon" type="image/png" href="popup-icons/favicon.png"/>
```
Comment (lenadgit) [inline submissions/Shrai-dev/html-css-popup/index.html:8]: The single tags should be without slashes "/" (according to HTML5 specification)

**Pull request merged**